docs/synapse/devguides/devops_cell.rst

```diff
@@ -0,0 +1,89 @@
+Cell Operations
+===============
+
+As detailed in :ref:`dev_architecture`, the ``Cell`` implements a number of core management functionalities,
+and is therefore used as the base class for Synapse applications.  It is also recommended to implement ``Cell`` for custom services.
+
+.. _devops-cell-config:
+
+Configuring a Cell Service
+--------------------------
+
+A Cell has a set of base configuration data that will also be inherited by any implementations:
+
+- ``dirn``: The storage directory for the Cell which is a required argument for service startup.
+- ``telepath``: Optional override for the Telepath URL to listen on.
+- ``https``: Optional override for the port to bind for the HTTPS/REST API.
+- ``name``: Optional additional name to share the service as.
+
+The Cell class also specifies configuration variables in ``confdefs``:
+
+- ``auth:passwd``: Optional bootstrapping for the root user password.
+- ``nexslog:en``: Optional enablement of Nexus logging (most custom Cell implementations will not set this).
+
+Cell implementations can extend the configuration variables available by specifying them in
+``confdefs`` in the Cell subclass.  The variable names can be namespaced using colons, for example ``mysvc:apikey``.
+
+Depending on deployment requirements, a combination of methods can be used for loading the configurations into the Cell.
+
+Config File
+***********
+
+A Cell has one optional configuration file, ``cell.yaml``, that may be located in the root Cell directory.
+The format of this file is YAML, and variable names are specified without alteration, for example::
+
+    ---
+    auth:passwd: secret
+    mysvc:apikey: 720a50f9-cfa1-43a9-9eca-dda379ecd8c5
+    ---
```

Comment:
I think the end of this block should be `...` not `---`

Reply:
fixed